### PR TITLE
Added Smallfile Dockerfile

### PR DESCRIPTION
**Pull request by chawlanikhil24**

### CODE REVIEW

File: smallfile/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM centos/tools
+
+USER root
+WORKDIR /root/
+RUN yum install git epel-release python-pip -y
```
Comment (jtaleric) [inline smallfile/Dockerfile]: I see pip installed, are you using it in the script you are writing?

Comment (chawlanikhil24) [reply smallfile/Dockerfile]: Yupp, pip is used to install redis python library along with elastic-search and numpy for post-data processing . The changes are existing on my local machine. Will be pushing them soon

Comment (jtaleric) [reply smallfile/Dockerfile]: Can we not install via yum instead of pip, that way it is using similar packages as the other workloads?

Comment (chawlanikhil24) [reply smallfile/Dockerfile]: Yupp, I just experimented with ` yum install python2-redis python2-numpy python2-elasticsearch -y` . This command can work as a  replacement for pip. I'll remove the further need for python-pip now.

**Pull request merged**